### ISSUE TITLE
WAC filter and search does not display results on High Availability

### ISSUE THREAD

**Issue by mgraves1981** [MicrosoftDocs/Windows-Admin-Center-Ideas-and-Feedback#78]
**Gateway Version**: 1.3.2111.01001
**Extension Version** that's been used if it applies

**To Reproduce**
Try to apply a filter or search for a name. Comes back as a white page. 

**Expected behavior**
I am expecting to see results if a filter is set or searching for a device by name. 

**Screenshots & Additional context**
![image](https://user-images.githubusercontent.com/94481948/142048380-70fb3e7f-2698-4871-95d1-221647e2ee0f.png)
![image](https://user-images.githubusercontent.com/94481948/142048427-6413881b-cce6-47d9-a8f4-b0c026584c71.png)
![image](https://user-images.githubusercontent.com/94481948/142048479-0da885ff-4e64-4c88-b5fc-7602fdb483ce.png)

**Comment by tddimitrov** [comment]
#61, #68, #79 - same issue...

**Comment by ahokanso** [comment]
Same behavior
Version  2110
Build  1.3.2111.01001
OS Windows Server 2019
200+ connection in Shared Connections so other admins don't have to populate there sessions in WAC.
Have tried to uninstall, Install and re-import connections to resolve this and this has not resolved the issue.

**Comment by trungtran-msft** [comment]
Yes, the search/filter among other things is fixed in the public preview for 2110.2. You can read about it and download the build here: https://techcommunity.microsoft.com/t5/windows-admin-center-blog/windows-admin-center-version-2110-2-is-now-in-public-preview/ba-p/3263839

**Comment by yanuoma** [comment]
Thanks for reporting this, @mgraves1981, and apologies for the delay in our triage cycle.

We took another look at this and believe it has been addressed:

- A maintainer confirmed in this thread that the search/filter on the Server Connections list (the white page you saw when filtering or searching for a device by name) was fixed in the Windows Admin Center **2110.2** public preview (https://techcommunity.microsoft.com/t5/windows-admin-center-blog/windows-admin-center-version-2110-2-is-now-in-public-preview/ba-p/3263839).
- The connection-list filtering/search behavior has since been reworked and lives in the current Gateway v2 shell, where filter/search is fully functional.
- This should be in WAC build 2110.2 and later.

If you can still reproduce on a current build, please reply here and we'll reopen — happy to dig in further with a fresh repro.

Closing for now. Thanks again for taking the time to write this up. 🙏

— The Windows Admin Center team
